default.nix

```diff
@@ -5,32 +5,42 @@ with lib;
 
 rec {
   qemu = pkgs.qemu.override {
+    guestAgentSupport = false;
+    numaSupport = false;
+    seccompSupport = false;
```

Comment:
This one you might actually want to keep, to layer on `-sandbox ...`. Unless you have apparmor/selinux already...

Reply:
It's not exactly identical, and having both would be preferable, but you do want at least one of them, just in case (qemu is a big blob of C, after all). There is some overlap, in that you can restrict e.g. network access with apparmor or seccomp.